pygmt/src/choropleth.py

```diff
@@ -0,0 +1,20 @@
+def choropleth(self, data, fillcol, cmap=True, **kwargs):
+    """
+    Plot a choropleth map from a :class:`geopandas.GeoDataFrame` object.
+
+    Parameters
+    ----------
+    data : :class:`geopandas.GeoDataFrame`
+        The geopandas dataframe containing the geometry and data to plot.
+    fillcol : str
```

Comment:
```suggestion
    column : str
```
I feel `fillcol` is not the correct name for this parameter. Would simply use `column`, as it is also used in e.g. [geopandas](https://geopandas.org/en/stable/docs/user_guide/mapping.html).